titus-server-master/src/main/java/com/netflix/titus/master/appscale/service/DefaultAppScaleManager.java

```diff
@@ -440,18 +438,15 @@ private String buildAutoScalingGroup(String jobId) {
     static String buildAutoScalingGroupV3(JobDescriptor<?> jobDescriptor) {
         JobGroupInfo jobGroupInfo = jobDescriptor.getJobGroupInfo();
         String jobGroupSequence = jobGroupInfo.getSequence() != null ? jobGroupInfo.getSequence() : DEFAULT_JOB_GROUP_SEQ;
-        List<String> parameterList = Arrays.asList(jobDescriptor.getApplicationName(), jobGroupInfo.getStack(), jobGroupInfo.getDetail(), jobGroupSequence);
-        return buildAutoScalingGroupFromParameters(parameterList);
+        AutoScalingGroupNameBuilder autoScalingGroupNameBuilder = new AutoScalingGroupNameBuilder();
+        String asgWithNoSequence = autoScalingGroupNameBuilder
```

Comment:
Can we add a comment here saying we're doing this because we are trying to map CWA name to how we expect Spinnaker/Atlas to emit CWM data?